.github/workflows/ci.yml

```diff
@@ -130,16 +130,22 @@ jobs:
          # ninja version to download. Default: 1.10.0
          version: 1.10.0
          dest: ${{ github.workspace }}/ninja_bin
+
+     - name: Add prebuilt llvm install dir to PATH
+       run: |
+         "${{ github.workspace }}/llvm-install-${{ env.LLVM_VER }}/bin\n" + (Get-Content $env:GITHUB_PATH -Raw) | Set-Content $env:GITHUB_PATH
+         Add-Content $env:GITHUB_ENV "CXX_COMPILER=${{ github.workspace }}/llvm-install-${{ env.LLVM_VER }}/bin/clang++.exe"
+
      - name: Configure CMake
        env:
          CMAKE_PREFIX_PATH: ${{ github.workspace }}/llvm-install-${{ env.LLVM_VER }}
-       run: cmake -B build -G Ninja -DFINAL_STAGE=1 -DCMAKE_CXX_COMPILER=clang.exe -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=jakt-install
+       run: cmake -B build -G Ninja -DFINAL_STAGE=1 -DCMAKE_CXX_COMPILER=clang -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=jakt-install
 
      - name: Build Jakt Stage 1 Selfhost
        run: cmake --build build
 
      - name: Test Jakt Stage 1
-       run: .\build\bin\jakttest.exe
+       run: .\build\bin\jakttest.exe --cpp-compiler $env:CXX_COMPILER
```

Comment:
Can't pass clang.exe here, there's a check that we're using clang-cl.exe :yakthonk:

Reply:
Or wait, that's for down below.

Reply:
Yeah I noticed, will check if we have clang-cl in the binaries.